Documentation/gettingstarted/k8s-install-openshift-okd.rst

```diff
@@ -151,7 +152,10 @@ Next, obtain Cilium manifest from ``cilium/cilium-olm`` repository and copy to `
 
    .. code-block:: shell-session
 
-       sed -i 's|image:\ registry.connect.redhat.com/isovalent/|image:\ quay.io/cilium/|g' "${CLUSTER_NAME}/manifests/cluster-network-06-cilium-00002-cilium-olm-deployment.yaml"``
+       sed -i 's|image:\ registry.connect.redhat.com/isovalent/|image:\ quay.io/cilium/|g' \
+         "${CLUSTER_NAME}/manifests/cluster-network-06-cilium-00002-cilium-olm-deployment.yaml" \
+         ${CLUSTER_NAME}/manifests/cluster-network-06-cilium-00014-cilium.*-clusterserviceversion.yaml
```

Comment:
```suggestion
         "${CLUSTER_NAME}/manifests/cluster-network-06-cilium-00014-cilium.*-clusterserviceversion.yaml"
```

Double-quotes for consistency with the line above?

Reply:
This was intentional because there is a `*`...